classes/class-kp-session.php

```diff
@@ -79,11 +86,12 @@ public function get_session( $order = null ) {
 		// Set session data from WC Session or order meta if we have any.
 		$this->set_session_data( $order );
 
-		// If we already have a Klarna session, and the session_country has changed since our last request, reset all our params and a new session will be created.
-		if ( null !== $this->klarna_session && kp_get_klarna_country( $order ) !== $this->session_country ) {
+		// If we already have a Klarna session, and the session_country or session_locale has changed since our last request, reset all our params and a new session will be created.
+		if ( null !== $this->klarna_session && kp_get_klarna_country( $order ) !== $this->session_country || get_locale() !== $this->session_locale ) {
```

Comment:
Should this if statement not have some extra parentheses? Since we are now mixing && and || i think it would be best to wrap `kp_get_klarna_country( $order ) !== $this->session_country || get_locale() !== $this->session_locale` with parentheses, so the if statement would be `if ( null !== $this->klarna_session && ( kp_get_klarna_country( $order ) !== $this->session_country || get_locale() !== $this->session_locale ) ) {`